DOCS/man/options.rst

```diff
@@ -6116,6 +6116,11 @@ them.
     Defines the size of an edge border (default: 32) to initiate client side
     resizes events in the wayland contexts with touch events.
 
+``--wayland-ime=<yes|no>``
+    Enable keyboard input via an active input method (IME) (default: yes). The
+    input popup window, if there is any, is always positioned at the top left of the
+    window.
```

Comment:
Does this need to be an option? I believe IME support is always on in windows.

Reply:
Since this is always enabled (not enabled/disabled based on if a console prompt is being shown), it affects keybindings. I added an option in case someone has their IME configured in a way that this would inconvenient. E.g. they default to a language which requires composition, or the current language is shared between all applications, so hovering over the mpv window and attempting to press a keybinding would bring up the composition window instead. They can disable it if they don't care about advanced user input in scripts and don't want to configure it to default to a deactivated state on mpv.
Similar complaint for windows: https://github.com/mpv-player/mpv/issues/13666

Reply:
Sure that's fair. Only problem is that the current implementation doesn't work with option changes at runtime.

Reply:
It can now be enabled/disabled at runtime, but this only takes action on the next `text_input_enter`. To have it take effect immediately, I could probably check it during `vo_wayland_control`/`wayland_dispatch_events`

Reply:
Disabling `wayland-ime` at runtime has no effect for me. The ime still works.

Reply:
Even after unfocusing/refocusing the window, and WAYLAND_DEBUG=1 says we aren't calling enable after enter again?

Reply:
It works for me with just the `preedit_string` and `done` events.

Reply:
So, to be clear, the composition window is still accessible and the compositor is still sending `preedit_string`/`done` after we sent `disable` and never sent `enable`?

Reply:
OK now it works when I focus/unfocus.